### PR TITLE
Upstep LittlevGL to version 5.3

### PR DESCRIPTION
**Pull request by vanwinkeljan**
Upstep LittlevGL to version 5.3

### CODE REVIEW

**Comment by nashif** [comment]
whats new in 5.3?

**Comment by vanwinkeljan** [comment]
@embeddedt, @kisvegabor fyi

**Comment by embeddedt** [comment]
@nashif This gives a pretty good overview: https://github.com/littlevgl/lvgl/releases/tag/v5.3

There are also various bugfixes, as always.

**Comment by vanwinkeljan** [comment]
@nashif should we try to get this in 1.14?

File: lib/gui/lvgl/Kconfig [hunk 1]
```diff
@@ -310,6 +333,21 @@ config LVGL_FILESYSTEM
...
+
+if LVGL_MULTI_LANGUAGE
+config LVGL_NBR_OF_LANGUAGES
+	int "Number of languages"
+	default 1
+	help
+	  Number of languages to support
+endif
+
+#define USE_LV_MULTI_LANG       0               /* Number of languages for labels to store (0: to disable this feature)*/
```
Comment (pizi-nordic) [inline lib/gui/lvgl/Kconfig]: Could we move this out of the Kconfig file?

File: lib/gui/lvgl/Kconfig.objects [hunk 2]
```diff
@@ -94,6 +100,23 @@ config LVGL_OBJ_TAB_VIEW_ANIMATION_TIME
...
 
+config LVGL_OBJ_TILE_VIEW
+	bool "Tile view object"
+	default y
+	depends on LVGL_OBJ_PAGE
+	help
+	  Enable tile view object support
+
+if LVGL_OBJ_TILE_VIEW
+
```
Comment (pizi-nordic) [inline lib/gui/lvgl/Kconfig.objects]: Please remove this empty line.

**Review by MaureenHelm — APPROVED** [review]
Works well on mimxrt1060_evk

**Review by nashif — APPROVED** [review]

**Pull request merged**